### ISSUE TITLE
[SUGGESTION] New Items (as Dungeon exclusive loot)

### ISSUE THREAD

**Issue by NiCeDiCe90** [pokeclicker/pokeclicker#1771]
**Current Behavior**
Currently we struggle with the balancing of Dungeon loot. On one side we want to make dungeons more rewarding but on the other side we make other mechanics basically useless because it's pretty easy to farm some of the items that were locked behind quests or the farm.

**Improved Behavior**
Add some unique Dungeon items that help on your journey to have a bigger item pool for dungeons.

**Detailed Description**
First off we need a way to sell these items. They are like the underground treasures, but are sold for other ressources.
We could also make some NPCs that trade these items for other/rare rewards like the reveal glass or something like that. So it might be worth to keep some of them instead of selling them for ressources.

These could be some of the items:

![grafik](https://user-images.githubusercontent.com/82889773/149530417-09f761e4-53fe-4d11-87b4-3ffd047d58cc.png) tropical shell - 200 DT
![grafik](https://user-images.githubusercontent.com/82889773/149530464-334050f2-f0bb-43b9-87eb-e2b5a1af96ac.png) beach glass - 500 DT
![grafik](https://user-images.githubusercontent.com/82889773/149531260-3d0072a0-e58b-4c3a-a241-e6afac44f3f5.png) marble - 1000 DT
![grafik](https://user-images.githubusercontent.com/82889773/149531415-6f381a20-cc26-4262-aec2-48bc3798f2ed.png) chalky stone - 50 BP
![grafik](https://user-images.githubusercontent.com/82889773/149531503-2159477d-f44e-4498-ac88-16a85eef31e9.png) pearl - 100 BP
![grafik](https://user-images.githubusercontent.com/82889773/149531557-c997fee1-198c-4778-9b81-e4ef285d97ae.png) big pearl - 200 BP
![grafik](https://user-images.githubusercontent.com/82889773/149531657-83fb75a3-e2a2-4a98-b7a8-301f79219b94.png) pearl string - 500 BP
![grafik](https://user-images.githubusercontent.com/82889773/149530506-031570da-aacd-4745-8717-d9abc05a1b0e.png) Star dust - 5 Diamonds
![grafik](https://user-images.githubusercontent.com/82889773/149530540-1bc008f9-4ce6-47df-920f-719c969a5478.png) strange souvenir - 10000$
![grafik](https://user-images.githubusercontent.com/82889773/149531921-939e5ad5-9c81-49be-94e2-0349c0242f42.png) polished mudball - 300 FP
![grafik](https://user-images.githubusercontent.com/82889773/149531462-87b720f6-2293-4db6-9cf2-84d1368544d6.png) tiny mushroom - 500 FP
![grafik](https://user-images.githubusercontent.com/82889773/149531597-111168f7-b3d0-4fc4-a5fe-778755caec1f.png) big mushroom - 1000P
![grafik](https://user-images.githubusercontent.com/82889773/149530615-d433d176-08eb-4b30-b020-47b335b4baca.png) balm mushroom - 2500 FP
![grafik](https://user-images.githubusercontent.com/82889773/149531359-b7ab2cf8-45ec-4d2b-9e52-39204c2b7e40.png) Comet shard - 2500 DT
![grafik](https://user-images.githubusercontent.com/82889773/149531787-e8f2b022-e63e-4b0b-a11d-d72678aa6e81.png) Nugget - 25000$
![grafik](https://user-images.githubusercontent.com/82889773/149531709-183e8135-5a9e-4b4e-ad0d-19b9255a8fff.png) Big Nugget - 50000$

**Comment by Ultima1990** [comment]
This is a neat idea. The problem is a problem we are having in general across the game. A lack of items. Not just dungeon items. Everywhere items. Items to sell in money shops, qp shops, bp shops, everywhere. Things that would make these currencies valuable. What we have here is a list of items that you can turn into currencies that have no real value atm. Aside from the DT ones which will probably quickly become worthless with the way DT costs (and gains) scale up across the game and will continue scaling up into Alola and Galar. 200 DT will be nothing.

So many problems would be solved if we come up with a series of items that people would want to buy (preferably repeatedly (_cough_ consumable held items _cough cough_)). Then we can add more ways to get the currencies, in which case these valuable dungeon drops would make more sense. Yes we have some great big one time currency sinks right now, Meloetta and Furfrou, but that's not a sustainable long term solution to this issue.

Back on this idea, we'd probably need a bunch of separate "treasure trader" or whatever npcs for each region that will buy the same items for every increasing prices so the value of these items can keep up with the ingame inflation. Like, the Kanto trader would buy the Tropical Shell for 200, but the Johto one would buy it for 400 and the Hoenn one would buy it for even more. To ensure the items don't become useless in later regions. Something like that.

**Comment by NiCeDiCe90** [comment]
You are absolutely right. I was thinking about reworking the idea #1570 and post it again. If we have 6 currencies and 6 vitamins and additionally consumable items tied to the party system. Could be a lot to buy in the future.

The idea of making the items worth more in higher regions has the downside that it's more easy to farm them if you go back to Kanto dungeons.
But we always could do a slight recolor of some of them and reintroduce them.
A blue Big Mushroom for example:
![bigmoosh](https://user-images.githubusercontent.com/82889773/149566443-cca56b03-244d-40a1-980b-2d243d99bbe1.png)

**Comment by jk13pclick** [comment]
I think I would like to see more items added now, irrespective of the fact that they will just inflate currency values. Consumable held items (or any kind of consumable for that matter) are a good end-goal for the item situation. However, adding "useless" items (which we can give better uses down the road) is probably not a bad idea for a temporary solution for Dungeon Loot rebalancing. Nor is it a particularly challenging idea to implement - it would probably just require some reworking of `Amount.ts` to allow for items to be used as currencies as well. 

Balancing for all item types in shops is necessary anyways as scaling is bad for all shops with fixed item prices. That is a problem that goes beyond Dungeon Loot - Dungeon Loot has just further exposed the flaw in a fixed-price system.
